apiserver/httpcontext/model.go

```diff
@@ -79,8 +79,11 @@ type modelKey struct{}
 // attempt is made to validate the model UUID; QueryModelHandler and
 // BucketModelHandler does this, and ControllerModelHandler should always be
 // supplied with a valid UUID.
-func RequestModelUUID(req *http.Request) (string, bool) {
-	if value := req.Context().Value(modelKey{}); value != nil {
+func RequestModelUUID(ctx context.Context) (string, bool) {
+	if ctx == nil {
+		return "", false
+	}
```

Comment:
Remove me, this isn't required. Just panic and die.

Reply:
Otherwise, every function would require this and this is just not the case.

Reply:
agreed, nil context is a programming error. If something asks for a context, it needs a context.